### PR TITLE
new line added

### PR DESCRIPTION
**Pull request by sunny-sk**
need to review  this code

### CODE REVIEW

File: a.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 const x ='sunny';
 console.log(x.length);
 
-let p = 'i am going to create new pull';
\ No newline at end of file
+let p = 'i am going to create new pull';
+let z = 'new page added'
+let x  = "this is added new line in the config";
```
Comment (sunny-sk) [inline a.js]: remove this

File: a.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 const x ='sunny';
 console.log(x.length);
 
-let p = 'i am going to create new pull';
\ No newline at end of file
+let p = 'i am going to create new pull';
+let z = 'new page added'
```
Comment (sunny-tft) [inline a.js]: rwea sdlkfjdso ]

**Review by sunny-tft — COMMENTED** [review]
sdrfdsgsdfg

File: a.js [hunk 3]
```diff
@@ -1,6 +0,0 @@
-const x ='sunny';
-console.log(x.length);
-
-let p = 'i am going to create new pull';
```
Comment (sunny-tft) [inline a.js]: why did you remove this

File: a.js [hunk 4]
```diff
@@ -1,4 +0,0 @@
-const x ='sunny';
-console.log(x.length);
```
Comment (sunny-tft) [inline a.js:2]: sedfxfvxcv

**Review by sunny-tft — CHANGES_REQUESTED** [review]
